### ISSUE TITLE
Add support for Android-x86. Possible?

### ISSUE THREAD

**Issue by doqfgc** [rivafarabi/deckboard#518]
first of all let me get out of the way that I absolutely love this concept and platform to death. okay.

I've been a Deckboard user for all of about four months. It's an incredible tool that I use on stream days using my ZTE Axon M in tent mode. It's 99% of the Stream Deck experience and does exactly what I wanted to do, to the point that the purchase of Pro that I made within 5 minutes of opening the free app is far and away without regret.

I recently made changes to my workflow, implementing a rather large touchscreen in my desk to use as a macro board for digital art, among other things. Unfortunately, Deckboard (and Pro) do not run on this setup-- an old upcycled Lenovo Flex 3 that I dismantled a while back-- the apps just crash with the "Deckboard stopped working" system message. I am pretty sure this is specific to Android-x86, even with Native Bridge enabled, as I don't have this issue on the aforementioned Axon M.

In conclusion: is it possible to support android-x86?

**Comment by Hexality** [comment]
Android x86 arch framework has been discontinued by google. You can use this on a x86 version of android by going to settings and enabling arm support.
